database/download.go

```diff
@@ -15,7 +24,8 @@ type Download struct {
 	UserID    primitive.ObjectID `bson:"user_id,omitempty" json:"userId"`
 	SkylinkID primitive.ObjectID `bson:"skylink_id,omitempty" json:"skylinkId"`
 	Bytes     int64              `bson:"bytes" json:"bytes"`
-	Timestamp time.Time          `bson:"timestamp" json:"timestamp"`
+	Created   time.Time          `bson:"timestamp" json:"timestamp"`
```

Comment:
Yes, you're right. I initially left it like this for development purposes (I didn't want to have to go over the records in the DB and update them manually) and then I forgot to change it.

I'll rename the `bson` fields to match the properties but for the moment I won't touch the `json` ones as that will change the API and that's not needed right now because we're not reporting the `updatedAt` time there.